client/src/components/Monument/Monument.js

```diff
@@ -255,6 +263,25 @@ export default class Monument extends React.Component {
         )
     }
 
+    renderInscription() {
+        const { monument } = this.state;
+
+        let inscription;
+        if (monument.inscription) {
+            inscription = (
+                <div>
+                    <b>Inscription:</b> {this.formatInscription(monument.inscription)}
```

Comment:
Don't use `<b>` it's deprecated (although there's `<strong>` now which is apparently ok to use), you should use CSS for bold: `font-weight: bold`, although bootstrap provides some convenient classes that you can use i.e.
```html
<span className="font-weight-bold">Inscription</span>
```

Reply:
I shall do that.

Can you explain what a `<span>` tag does in a very noob way? :)